presto-hive/src/main/java/com/facebook/presto/hive/HivePageSinkProvider.java

```diff
@@ -28,6 +28,7 @@
 import com.google.common.util.concurrent.ListeningExecutorService;
```

Comment:
I think commit message needs to be a bit improved. I do not know what this commit is about without reading the code.